chemicaltoolbox/autodock_vina/docking/docking.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="docking" name="Docking" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.1.2">autodockvina</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <description> - Tool for running docking with Autodock Vina</description>
+    <command interpreter="python"><![CDATA[
+        docking.py -l "$ligand" 
+                   -r "$receptor" 
+                   -b "$box" 
+                   -p "$file_output1.files_path" 
+                   -f "$file_output1" 
+                   -g "$file_output2"
+    ]]></command>
+    <inputs>
+        <param type="data" name="ligand" format="pdbqt" label="Ligands: " help="No data? See tip below" />
+        <param type="data" name="receptor" format="pdbqt" label="Receptor: " help="No data? See tip below" />
+        <param type="data" name="box" format="txt" label="Binding site coord.: " help="No data? See tip below" />
+    </inputs>
+    <outputs>
+        <data name="file_output1" format="pdbqt" label="#echo (os.path.splitext (str($ligand.name))[0]) + '_-_' + (os.path.splitext (str($receptor.name))[0])#.pdbqt" />
+        <data name="file_output2" format="log" label="#echo (os.path.splitext (str($ligand.name))[0]) + '_-_' + (os.path.splitext (str($receptor.name))[0])#.log" />
```

Comment:
I removal log data type and we can use as simple text file, but I need this in tool output, because we use the informations about scoring that are stored in this file. More tools will be developed and they will need this information.
These tools are used for the supporting the [drugdesign](https://github.com/rodrigofaccioli/drugdesign) project, which being developed with [rodrigofaccioli.](https://github.com/rodrigofaccioli) We could talk more about our future plans if you think interesting.